packages/ra-language-english/src/index.ts

```diff
@@ -118,6 +118,7 @@ const englishMessages: TranslationMessages = {
             unsaved_changes:
                 "Some of your changes weren't saved. Are you sure you want to ignore them?",
             yes: 'Yes',
+            placeholder_data_warning: 'Could not fetch the requested data',
```

Comment:
I'd be more explicit with this message, mentioning lost connection to the server

Reply:
I changed it to `'Could not fetch: lost connection to server'`. Is it better?